client/setup.js

```diff
@@ -68,6 +75,18 @@ var Setup = function () {
     return select;
   }
 
+  function durationSelect(){
+    let duration = document.createElement("select");
```

Comment:
the duration element should have its default value set to 5 minutes

Reply:
Updated default to 5 min.